Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -168,18 +177,14 @@ std::map<std::string, std::string> Stitch1DMany::validateCommonInputs() {
     errors["InputWorkspaces"] = "At least 2 input workspaces required.";
 
   // Check that all the workspaces are of the same type
-  if (!m_inputWSMatrix.empty() && !m_inputWSMatrix[0].empty()) {
```

Comment:
The new `ADSValidator` dealt with the problem I mentioned before, so it isn't needed anymore.